src/audio/copier/copier.c

```diff
@@ -528,13 +528,49 @@ static int do_conversion_copy(struct comp_dev *dev,
 	return 0;
 }
 
+static int copier_copy_to_sinks(struct copier_data *cd, struct comp_dev *dev,
+				struct comp_buffer __sparse_cache *src_c,
+				struct comp_copy_limits *processed_data)
+{
+	struct comp_buffer __sparse_cache *sink_c;
+	struct list_item *sink_list;
+	struct comp_buffer *sink;
+	int ret = 0;
+
+	list_for_item(sink_list, &dev->bsink_list) {
```

Comment:
Large code block - can we have an inline comment on what we are doing for each list item.

Reply:
this code is actually moved out from do_multi_endpoint_module_copy, it is module copy part, I am adding simple comments for explain it.